### PR TITLE
Adds shared volume

### PR DESCRIPTION
**Pull request by JJ**
And documentation on how to use it.

These are very minor changes but add to usability.

### CODE REVIEW

**Comment by JJ** [comment]
(I know I have the privs to merge this, but maybe someone else wants to check it out)

File: README.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ Simply running a container with the image will launch a Perl 6 REPL:
 You can also provide perl6 command line switches to `docker run`:
 
     $ docker run -it rakudo-star perl6 -e 'say "Hello!"'
+	
+or run your own files using the container's interpreter
+
+	$ docker run -t -v `pwd`:/home/perl6/app rakudo-star /home/perl6/app/ls.p6
+
+where we mal the local file ls.p6 (in this directory) to the remote
```
Comment (alexm) [inline README.md]: What does `mal` mean here? It sounds like a typo, but I may be wrong.

**Review by alexm — COMMENTED** [review]
I tried this on an older docker image and it works fine:

```
docker run -t -v `pwd`:/home/perl6 rakudo-star perl6 /home/perl6/ls.p6
```

Is the volume really needed?

File: README.md (hunk 1 above)
Comment (JJ) [reply README.md]: totally typo. Should be map. Will fix that.

Comment (JJ) [reply README.md]: Volume is not really needed. It's just a convenient place to put scripts. You can create new volumes with -v all the time.

**Review by alexm — APPROVED** [review]

**Comment by tianon** [comment]
As I noted over in https://github.com/docker-library/official-images/pull/2599, I'm having a hard time understanding what's added by having an explicit `VOLUME` instruction in the `Dockerfile` here -- if `-v` is used on the `docker run` line, a volume or bind mount is created already, and no pre-defined `VOLUME` is actually necessary.

**Comment by JJ** [comment]
No, not really. It's just a convention,

**Comment by tianon** [comment]
I think the description in https://github.com/docker-library/official-images/pull/2437#issuecomment-266578827 is decent context on why this `VOLUME` doesn't seem terribly appropriate (and could actually be actively harmful to users, while the absence of the explicit `VOLUME` doesn't prevent anything that's newly documented here):

> The general rule _we_ follow is that you should have a volume for where the application stores data (so `$SONARQUBE_HOME/data` is uncontestably appropriate), but configuration is almost never appropriate in a volume (unless it's specifically managed by the application itself at runtime and _never_ supplied/managed manually by users, or there is an easy way to override the default in the image).
> 
> The reason for this is that, once defined, Docker has no means of "`UNVOLUME`" -- thus, once a volume, always a volume.

**Comment by JJ** [comment]
OK, you have a point here.

**Pull request merged**